cmake/extensions.cmake

```diff
@@ -1586,9 +1586,12 @@ endfunction()
 # manager configurations.
 function(add_partition_manager_config config_file)
   get_filename_component(pm_path ${config_file} REALPATH)
+
+  # Add entry to PM_SUBSYS on the format
+  # "<abs path to pm.yml> <build dir outut folder>"
   set_property(
     GLOBAL APPEND PROPERTY
     PM_SUBSYS
-    ${pm_path}
+    "${pm_path} ${CMAKE_CURRENT_BINARY_DIR}"
```

Comment:
I noticed, this will be split using a regex later.
I haven't been through all parts of existing code, but is the values on property PM_SUBSYS alsways double like here ?
Cause in that case, we could considered to do double lists.

Reply:
We can do a double list (do you mean a map/dict? Because we need to look up on a per path basis). But the problem is that the path includes symbols not suited for a cmake variable.

Reply:
I simply meant two lists, similar to:
```
set_property(
    GLOBAL APPEND PROPERTY
    PM_SUBSYS_PATH ${pm_path}
)
set_property(
    GLOBAL APPEND PROPERTY
    PM_SUBSYS_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR}
)
```
see following comment here: https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/2122#discussion_r409436940

Reply:
Got it! Will update